packages/cli/src/prebuild.ts

```diff
@@ -589,62 +639,67 @@ export const prebuild = async (options: {
     const pageMeta = pageData.page.meta;
     const favIconAsset = assets.get(projectMeta?.faviconAssetId ?? "");
     const socialImageAsset = assets.get(pageMeta.socialImageAssetId ?? "");
+
+    // MARK: - TODO: XML GENERATION
     const pageExports = `/* eslint-disable */
-/* This is a auto generated file for building the project */ \n
+      /* This is a auto generated file for building the project */ \n
```

Comment:
we don't run prettier for user code, would be weird indent

Reply:
Fully unreadable for me otherwise. Also additional cognitive load to think about writing string code being looking good.